cmd/seednode/main.go

```diff
@@ -49,6 +50,12 @@ VERSION:
 			"`0, 10230, 15670` or range of ports such as `5000-10000`",
 		Value: "10000",
 	}
+	restApiPortFlag = cli.IntFlag{
```

Comment:
we should be able to define the interface as well

Reply:
done